### PR TITLE
PBI238 - Machine login

### PR DESCRIPTION
**Pull request by bsonntag**
Adds `.machineLogin()` method to `VideoCallClient`.

### CODE REVIEW

**Comment by bsonntag** [comment]
@DanielMor and @Mickael24 please review.

File: lib/video-call-client.js [hunk 1]
```diff
@@ -18,46 +18,75 @@ function VideoCallClient(SocketIOClient, WebRTC, url, config, constraints) {
...
   client.machines = [];
   client.login = login;
+  client.machineLogin = machineLogin;
   client.startCall = startCall;
   client.createVideoMessage = VideoMessage.bind(null, client.webRtc);
   client.disconnect = disconnect;
 
   return client;
 
   function login(options) {
```
Comment (Mickael24) [inline lib/video-call-client.js]: dont receive url?

File: lib/video-call-client.js [hunk 2]
```diff
@@ -18,46 +18,75 @@ function VideoCallClient(SocketIOClient, WebRTC, url, config, constraints) {
...
 
-      _setupSocket(socket);
+      _setupSocket(socket, _createHandlers(clientType));
 
       client.emit('connected');
     }
   }
 
-  function _setupSocket(socket) {
+  function _createHandlers(clientType) {
```
Comment (Mickael24) [inline lib/video-call-client.js]: you can put this handlers logic in file Handler.

File: lib/video-call-client.js (hunk 1 above)
Comment (bsonntag) [inline lib/video-call-client.js]: `url` is a parameter of `VideoCallClient`.

**Comment by bsonntag** [comment]
@Mickael24 :eyes:

**Comment by Mickael24** [comment]
👍

**Comment by bsonntag** [comment]
@DanielMor bump

**Pull request merged**